src/index.css

```
@@ -0,0 +1,37 @@
+:root {
+  font-family: 'Inter', system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', sans-serif;
```

Stylelint (17.2.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)